src/dlgTriggerEditor.cpp

```diff
@@ -4613,7 +4655,7 @@ void dlgTriggerEditor::slot_var_selected(QTreeWidgetItem *pItem)
     if (varType == LUA_TTABLE || varType == LUA_TFUNCTION)
     {
         // Edbee doesn't have a readonly option, so I'm using setEnabled
-        mpSourceEditorEdbee->setEnabled(false);
+        // mpSourceEditorEdbee->setEnabled(false);
```

Comment:
Tables (once they are populated ?) and Functions CAN/SHOULD not be edited in the variables tree - only displayed IIRC...!